NYT360Video.podspec

```diff
@@ -1,24 +0,0 @@
-Pod::Spec.new do |s|
-    s.name             = 'NYT360Video'
```

Comment:
did we need to remove this?

Reply:
I'm not sure why this is showing up in the diff still, this whole podspec was deleted (it was a remnant from the old project, and at some point we made the new `TVOS360Video.podspec`).

Reply:
but should it be deleted as part of this PR, i would assume it was deleted previously